sample-chat/src/main/java/com/quickblox/sample/chat/ui/activity/ChatInfoActivity.java

```diff
@@ -31,24 +35,38 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
 
-        qbDialog = (QBChatDialog) getIntent().getSerializableExtra(EXTRA_DIALOG);
+        actionBar.setDisplayHomeAsUpEnabled(true);
         usersListView = _findViewById(R.id.list_login_users);
+        qbDialog = (QBChatDialog) getIntent().getSerializableExtra(EXTRA_DIALOG);
 
-        actionBar.setDisplayHomeAsUpEnabled(true);
+        updateCurrentDialog();
+    }
+
+    private void updateCurrentDialog() {
```

Comment:
incorrect method name, "update" means that you're putting changes to server, so it should be "load"/"get"/"refresh" depending on overall code style